release-2.3.10.yml

```diff
@@ -1,13 +1,18 @@
 project: ENTSBT
 key: ENTSBT-1060
 version: 2.3.10
+previousVersion: 2.3.6
 schedule:
   # Release date scheduled. It will also be used to populate the description to be created for each Jira stakeholder: component or starter owner
   release: 2021-05-31
```

Comment:
Shouldn't we modify this date? It's already passed....

Reply:
That's the date the file was created with. I don't have another date to put there, either.